admin-dev/themes/new-theme/js/vue/components/DragFileUpload.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div />
+</template>
+
+<script>
+  export default {
+    name: 'DragFileUpload',
+    props: {
+      dropzoneSelector: {
+        type: String,
+        default: '',
+      },
+    },
+    mounted() {
+      document.querySelector(this.dropzoneSelector).addEventListener('dragover', (e) => {
```

Comment:
Better implementation of dragover and drop events.
It's against vuejs rules to use document.querySelector
https://codepen.io/pasoevi/pen/ooOrpo

Reply:
nice exmaple. I've tried to use @drop previously, but it wasn't working. Now i see it was because i must prevent dragover default even tho i only need the drop.